SETUP.md

```diff
@@ -183,6 +183,15 @@ To continuously generate assets as you make changes to files (less, coffeescript
 $ yarn run watch
 ```
 
+## Email
+
+You can watch emails being sent from `osu-web` by watching the `storage/logs/laravel.log` file. Make sure the `MAIL_DRIVER` is set to `log`.
+
+## Use the API from osu!
+
+To connect from osu!(lazer) via the API offered by osu-web, you need to create an OAuth client from your user settings. Then, open the `oauth_clients` table and set the `personal_access_client` and `password_client` columns to `1`.
```

Comment:
pretty sure that entire thing could be done using artisan passport commands (would require #6945 to be fixed in the first, though)